src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -428,7 +428,7 @@ export const DatasetBuilderView = ({ datasetId }: DatasetBuilderProps) => {
   return datasetDetails.status === 'Ready'
     ? h(FooterWrapper, [
         h(TopBar, { title: 'Preview', href: '' }, []),
-        h(DatasetBuilderHeader, { name: datasetDetails.state.name }),
+        h(DatasetBuilderHeader, { name: datasetDetails.state.name, datasetId }),
```

Comment:
If `datasetDetails` contains `datasetId` (which it seems like it should) then maybe `DatasetBuilderHeader` should accept `datasetDetails` instead of `name` and `dataseId`.

Reply:
(The response already included it, the type didn't - will update)